### ISSUE TITLE
Re-enable component based A11Y tree snapshot tests

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#3814]
Scope
---
The old Puppeteer A11Y tree snapshot tests were deactivated because of deprecation and distracting outputs. But with the Playwright ARIA snapshot testing functionality, this testing feature gets interesting again and should be implemented on component level e2e testing.

Out of Scope
---

Notes  
---
- https://playwright.dev/docs/aria-snapshots
- This kind of test is already integrated into the AI-Widget template
- Search for redundant ARIA attribute e2e testing and remove it

Acceptance criteria
---
- [ ] Playwright ARIA snapshot testing is integrated

Subtasks
---
- [ ] Enhance Component A11Y Docs with additional Automated Test table row
